common4j/src/main/com/microsoft/identity/common/java/providers/nativeauth/NativeAuthConstants.kt

```diff
@@ -0,0 +1,10 @@
+package com.microsoft.identity.common.java.providers.nativeauth
+
+object NativeAuthConstants {
```

Comment:
We have a duplicate class with the same purpose: this one and `NativeAuthGrantType`. Let's remove one of the two.